### ISSUE TITLE
INPC007 don't warn for sealed class with no mutable properties

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/PropertyChangedAnalyzers#37]
```cs
namespace RoslynSandbox
{
    using System.ComponentModel;

    public sealed class Foo : INotifyPropertyChanged
    {
        public Foo(int value )
        {
            this.Value = value;
        }

        public event PropertyChangedEventHandler PropertyChanged;

        public int Value { get; }

        public int Squared => this.Value * this.Value;
    }
}
```
